.env.example

```diff
@@ -18,8 +18,10 @@ JWT_VERIFY_EMAIL_EXPIRATION_MINUTES=10
 
 # SMTP configuration options for the email service
 # For testing, you can use a fake SMTP service like Ethereal: https://ethereal.email/create
+# SMTP_SECURE Use TLS or upgrade later with STARTTLS
 SMTP_HOST=email-server
 SMTP_PORT=587
+SMTP_SECURE=false
```

Comment:
From [Nodemailer docs](https://nodemailer.com/smtp/):

> secure – if true the connection will use TLS when connecting to server. If false (the default) then TLS is used if server supports the STARTTLS extension. In most cases set this value to true if you are connecting to port 465. For port 587 or 25 keep it false

Observations:

1. Maybe add that full quote on the Joi description
2. Maybe set default state based on configured port on `src/config/config.js` if no `SMTP_SECURE` variable has been set on the environment

Reply:
I think the short description pretty much says the same as the docs, only short and precise.

I was trying to have `SMTP_SECURE` to be set by `SMTP_PORT` like this:

```javascript
switch (true) {
  case process.env.SMTP_PORT === 465:
    process.env.SMTP_SECURE = true;
    break;
  default:
    process.env.SMTP_SECURE = false;
    break;
}
```
But that's a **No-No**. Where do you suggest I do this?

On the other hand, I don't believe I have ever encountered, in the wild, a server using port 465.
The wast majority out there, is relying on `STARTTLS` to upgrade to a secured connection.
I could be wrong though :)

Reply:
# Yes. I was wrong.

*My server* is not listening on port 465 for some reason. Why did I think it should listen on 587 ???
Investigation.